src/components/LearnMore/LearnMore.tsx

```diff
@@ -78,6 +77,12 @@ const LearnMoreContainer = styled.article`
             font-size: 25px;
         }
     }
+
+    @media (max-width: 900px) {
```

Comment:
Please use/add helper
https://onja20.slack.com/archives/C01BXQRM6LV/p1627044969006300
https://codesandbox.io/s/media-queries-in-styled-components-typescript-forked-o4tt0